### ISSUE TITLE
[Feature Request] Recipe-less crafting.

### ISSUE THREAD

**Issue by Panzybear** [AngryBeaver/beavers-crafting#19]
Would love for players to be able to discover recipes through experimenting.

Example: Recipe is not known to player, but they add the correct ingredients into a crafting panel and attempt to craft without the recipe allows them to create the item and "learn" the recipe for future use. (Changing ownership to observer or how ever best to achieve this.)
Failure doesn't spend items, just generates a "no effect" text box.

**Comment by AngryBeaver** [comment]
Yes, i also had thought about it, but given the complexity and that players are fast bored out i didn't work on it. (playing divinitiy sin i know how fast it gets boring after some tries you just give up on finding out recipes, just my experience).

based on this i give it a two out of ten on my excitement scala to develop this feature.
-> i keep it in mind and play with the idea but, do not expect anything soon especially as i am only working on this at the weekends. the next two weekends i am probably still busy with the system independent adaption.

**Comment by Panzybear** [comment]
Good deal!
Thanks for the insight!
